File: Selenium/Activities/activity20.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;


public class activity20 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://www.training-support.net/selenium/javascript-alerts");
        System.out.println("The title of the page is: " +driver.getTitle());
        //Find the button to open a PROMPT alert and click it.
        driver.findElement(By.id("prompt")).click();
        //Switch the focus from the main window to the Alert box and get the text in it and print it.
        Alert prompt_alert = driver.switchTo().alert();
        System.out.println("The text in the prompt alert box is: " +prompt_alert.getText());
        //Type "Awesome!" into the prompt.
        prompt_alert.sendKeys("Awesome!");
        //Close the alert by clicking Ok
        prompt_alert.accept();
        driver.quit();
    }
}

File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.remote.webdriver import By

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:
    driver.get("https://www.training-support.net/selenium/javascript-alerts")
    print("Page title is: ", driver.title)
    #Find the button to open a CONFIRM alert and click it.
    driver.find_element(By.ID, "prompt").click()
    #Switch the focus from the main window to the Alert box and get the text in it and print it.
    prompt_alert = driver.switch_to.alert
    print(prompt_alert.text)
    #Close the alert once with Ok and again with Cancel.
    prompt_alert.send_keys("Awesome!")
    prompt_alert.accept()
    print("The alert is accepted")
    driver.find_element(By.ID, "prompt").click()
    prompt_alert1 = driver.switch_to.alert
    print(prompt_alert1.text)
    prompt_alert1.dismiss()
    print("The alert is dismissed")